requirements.txt

```diff
@@ -1,2 +1,3 @@
 MySQL-python==1.2.5
 pandas==0.16.2
+sailthru-client
```

Comment:
Add a version. Also, given the divergent natures of these scripts, each script directory should have its own requirements file. There's no reason you should install either of the existing requirements for this script.